build/yamls/antrea-node-init.yml

```diff
@@ -0,0 +1,42 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: antrea-node-init
+  namespace: kube-system
+  labels:
+    app: antrea-node-init
```

Comment:
All antrea managed resources have "app: antrea" labels so easy to select and clean up, maybe use the same app label and component: antrea-node-init?

Reply:
Done